scripts/preversion.sh

```diff
@@ -2,8 +2,9 @@
 
 set -eu
 
-# Check that tag signing works
-git tag --sign --message "Dummy Tag" dummy-tag
+# Check that tag creation works.
+# The tag is not currently signed because Jenkins does not have this set up.
+git tag --message "Dummy Tag" dummy-tag
 git tag --delete dummy-tag > /dev/null
```

Comment:
It should be possible to get the "jenkins-hypothesis" user to sign his tags, but it isn't essential so I've skipped that for now.